static/js/util.js

```diff
@@ -293,23 +295,31 @@ export function clean_user_content_links(html) {
         // Ensure that the title displays the real URL.
         let title;
         let legacy_title;
+
         if (url.origin === window.location.origin && url.pathname.startsWith("/user_uploads/")) {
             // We add the word "download" to make clear what will
             // happen when clicking the file.  This is particularly
             // important in the desktop app, where hovering a URL does
             // not display the URL like it does in the web app.
-            title = legacy_title = $t(
```

Comment:
This code looks like it applies to other places other than the image previews, so I'd be wary about making this large of a change without explaining why it is correct. At the least, the comment above should be updated to describe the current state of the code, as it appears to be out of date now. @timabbott might be able to chime in here.